src/renderer/entities/walletConnect/model/wallet-connect-model.ts

```diff
@@ -330,7 +340,8 @@ export const walletConnectModel = {
   $pairings,
   events: {
     connect,
-    disconnect,
+    disconnectCurrentStarted,
```

Comment:
`Disconnect current ...` feels that noun is missing, current topic/session or something else?